### ISSUE TITLE
BLE and Serial in same project?

### ISSUE THREAD

**Issue by AshMartian** [douglasjunior/AndroidBluetoothLibrary#15]
I was wondering if it were possible to use this library to use both BLE and Bluetooth Serial at the same time (at least for scanning)? 
My application needs to connect to Bluetooth LE, Serial and Wifi devices, 1 active connection, but needs support for all three methods of connection. From my initial research into this library it doesn't seem possible. Am I wrong? Can both libraries be used in the same project?

**Comment by douglasjunior** [comment]
You are right, today is no possible. But I guess that it is possible with a small change.

I will look at this in next weekend, ok?

**Comment by blandman** [comment]
Wonderful, thank you. Let me know if there’s anything I can test or do to help.

**Comment by mingdee** [comment]
I am having the same requirement - BLE and Classic scanning at the same time but just need to connect either LE or classic. Is the current library support this now?

**Comment by jjrodcast** [comment]
@mingdee I think you have to give priority to one of them if the devices doesn't support one use the other option.

**Comment by DaBs** [comment]
The problem becomes when you want to write actual payloads with BLE, seeing as characteristics are NOT made to tolerate byte streams.

**Comment by rizwan321** [comment]
Hi guys. i have some requirements which needs both Bluetooth medium Bluetooth Serial and BLE as well. 

like 

i have to get battery level => BLE
i have to update firmware => Classic bluetooth

i am very beginner in bluetooth my first question is can we connect to bluetooth in the same time ?
